files/common/var/lib/delphix-platform/os-migration

```diff
@@ -84,6 +84,28 @@ function perform_migration() {
 	chown -R postgres:postgres /mds ||
 		die "Failed chowning /mds"
 
+	#
```

Comment:
nit: we can add something like `echo "Fixing mountpoint permissions for domain0 datsets"` just so it's easier for someone reading the logs to have a grasp of where we are.

Reply:
Added